explorer/frontend/pages/_app.jsx

```diff
@@ -44,6 +46,7 @@ const App = ({ Component, pageProps }) => {
 
 	return (
 		<div className={styles.wrapper}>
+			<script dangerouslySetInnerHTML={{ __html: `window.appConfig = ${JSON.stringify(appConfig)};` }} />
```

Comment:
I am assuming the ``dangerouslySetInnerHTML`` is safe because it is trusted data? 
https://blog.logrocket.com/using-dangerouslysetinnerhtml-react-application/

Reply:
Yes, it should be safe enough in this case, since the data is injected into the page by the server and consists only of environment variables provided when running the Docker container.

It would need to be sanitized if it came from a user or any third-party source, to prevent malicious scripts from being injected.